cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1059,47 +1059,33 @@ pub struct ManifestContainerizeConfig {
     /// If `group`/`gid` is not specified, the default group and supplementary groups of the given `user`/`uid` in `/etc/passwd` and `/etc/group` from the container are applied.
     /// If `group`/`gid` is specified, supplementary groups from the container are ignored.
     #[serde(skip_serializing_if = "Option::is_none")]
-    user: Option<String>,
+    pub user: Option<String>,
     /// A set of ports to expose from a container running this image.
     /// Its keys can be in the format of:
     /// `port/tcp`, `port/udp`, `port` with the default protocol being `tcp` if not specified.
     /// These values act as defaults and are merged with any specified when creating a container.
-    /// **NOTE:** This JSON structure value is unusual because it is a direct JSON serialization of the Go type `map[string]struct{}` and is represented in JSON as an object mapping its keys to an empty object.
-    #[cfg_attr(
-        test,
-        proptest(
-            strategy = "proptest::option::of(proptest_btree_map_alphanum_keys_empty_map(10, 3))"
-        )
-    )]
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    exposed_ports: Option<BTreeMap<String, BTreeMap<(), ()>>>,
+    pub exposed_ports: Option<Vec<String>>,
```

Comment:
NB: Shouldn't this be a Set of sorts? At least that's what the original syntax meant to emulate.

Reply:
I've converted these both to `BTreeSet` in a separate commit which I won't squash.

There's no functional difference though because TOML has no notion of sets and they were getting de-duped anyway.